week-2/J-array-get-set/exercise.js

```diff
@@ -5,11 +5,11 @@
 */
 
 function first(arr) {
-  return; // complete this statement
+  return arr[0]; // complete this statement
 }
 
 function last(arr) {
-  return; // complete this statement
+  return arr[2]; // complete this statement
```

Comment:
Not quite. This will return the 3rd character of the string you pass to it. How can you ensure you always know the index of the last character in a string? Hint: you'll need to know how many characters are in the string...